webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -311,4 +315,38 @@ test.describe('Experiment List', () => {
     await row.experimentActionDropdown.pause.pwLocator.click();
     await expect.soft((await row.getCellByColumnName('State')).pwLocator).toHaveText('paused');
   });
+
+  test('Datagrid Bulk Action', async () => {
+    // should probably go last/before move
```

Comment:
you won't need to go last since you have multiple. just make sure this test gets its own experiment